src/main/java/budgetbuddy/model/transaction/Amount.java

```diff
@@ -40,6 +40,9 @@ public static boolean isValidAmount(long test) {
     public String toString() {
```

Comment:
Perhaps it might be better retaining the long value here, and perhaps use only string format as such:

`return String.format("%d.%02d", amount / 100, amount % 100);`

Makes it look a little simpler. `%02d` pads the cents such that the total length is 2.

Reply:
Indeed that's a much better way, I'll amend it.